components/SampleAddressInput/SampleAddressInput.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * @author Felix Hildebrandt <fhildeb>
+ */
+
+import { useContext } from 'react';
+import { NetworkContext } from '../../contexts/NetworksContext';
+
+enum AddressType {
+  UP = 'UP',
+  Asset = 'Asset',
+}
+
+interface Props {
+  inputId: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const SampleAddressInput: React.FC<Props> = ({ inputId, onChange }) => {
+  const { network } = useContext(NetworkContext);
+
+  const changeInputAddress = (type: AddressType) => {
+    let address;
+    if (type === AddressType.UP) {
+      if (network.name === 'MAINNET') {
+        // TODO: Update to Mainnet Profile
+        address = '0x9139def55c73c12bcda9c44f12326686e3948634';
+      } else if (network.name === 'TESTNET') {
+        address = '0x9139def55c73c12bcda9c44f12326686e3948634';
+      }
+    } else if (type === AddressType.Asset) {
+      if (network.name === 'MAINNET') {
+        // TODO: Update to Mainnet Asset
+        address = '0x6395b330F063F96579aA8F7b59f2584fb9b6c3a5';
+      } else if (network.name === 'TESTNET') {
+        address = '0x6395b330F063F96579aA8F7b59f2584fb9b6c3a5';
+      }
+    }
+
+    if (address) {
+      const inputElement = document.getElementById(inputId) as HTMLInputElement;
+      if (inputElement) {
+        inputElement.value = address;
+        onChange({
+          target: { value: address },
+        } as React.ChangeEvent<HTMLInputElement>);
+      }
+    }
```

Comment:
no need for that - this looks weird

Reply:
Where is the element with `{inputId}` class defined??

Let's try to not hijack the system by targetting dom elements like that. I'm sure there is another easiest way.